macrofusion.py

```diff
@@ -107,11 +107,11 @@
         # Use GPU for remapping.
         "use_gpu"               : ["--gpu",     True],
         # Correlation threshold for identifying control points (default: 0.9).
-        "corr_thres"            : ["--corr",    0.1],
+        "corr_thres"            : ["--corr",    0.9],
         # Remove all control points with an error higher than num pixels (default: 3).
-        "ctrl_pnt_thr"          : ["-t",        1],
+        "ctrl_pnt_thr"          : ["-t",        3],
         # Number of control points (per grid, see option -g) to create between adjacent images (default: 8).  
-        "num_ctrl_pnt"          : ["-c",        20],
+        "num_ctrl_pnt"          : ["-c",        8],
         # Scale down image by 2^scale (default: 1). Scaling down images will improve speed at the cost of accuracy.
         "scale_down"            : ["-s",        0],
```

Comment:
should we also change to default value?
```suggestion
        "scale_down"            : ["-s",        1],
```

Reply:
But there no access to change the value from the GUI and scale_down = 1 reduces quality?
I would guess users wants best quality without tinkering with the source code ;-)

Reply:
Makes sense, I just saw that you consistently set the default values from the comments to the actual config values, except this one, hence I thought for consistency reasons it would also make sense here. But I'm fine and agree with your argumentation ;)